docs/src/samples/publication-example-objects.yaml

```diff
@@ -9,10 +9,8 @@ spec:
   dbname: app
   target:
     objects:
+      - tablesInSchema: public
       - table:
```

Comment:
OK, so this will now fail, since tablesInSchema is incompatible with `table`, right? If so should the sample be renamed to reflect that it will fail?

Reply:
The example works perfectly. If you want a failing example here I have one:

```
apiVersion: postgresql.cnpg.io/v1
kind: Publication
metadata:
  name: publication-example-objects-fail
spec:
  cluster:
    name: cluster-example
  name: pub-objects-fail
  dbname: app
  target:
    objects:
      - tablesInSchema: public
      - table:
          schema: another_schema
          name: numbers_three
          only: true
          columns:
            - i
            - m
```